flyctl/app_config.go

```diff
@@ -25,7 +25,7 @@ const (
 type AppConfig struct {
 	AppName string
 	Build   *Build
-
+	// DockerBuildArgs map[string]string
```

Comment:
just nuke it 😉

Reply:
Oh dash, nuked now. Must get into the habit of labelling kipple comments.